### ISSUE TITLE
[Bug] [Windows] Semantic description read from timepicker twice

### ISSUE THREAD

**Issue by rachelkang** [dotnet/maui#1185]
### Description

Semantic description is read two times from timepicker control. It should only read once.

### Steps to Reproduce

1. Run MAUI sample on Windows
2. Navigate to Semantics page 
3. Turn on Narrator
4. Tab navigate to timepicker controls
5. Notice description text is read aloud twice from the control

Note: This used to also be the case for the datepicker control, but is no longer.

### Expected Behavior
Semantic description is read only once from timepicker control.

### Actual Behavior
Semantic description is read two times from timepicker control.

### Basic Information

- Version with issue:
- Last known good version:
- IDE:
- Platform Target Frameworks: 
  - iOS:  
  - Android:  
  - UWP:   
- Android Support Library Version: 
- Nuget Packages:
- Affected Devices:

**Comment by rachelkang** [comment]
This remains an issue

**Comment by XamlTest** [comment]
This issue **not repro** with Visual Studio Enterprise 2022 main 17.2.0 Preview 2.0 [32227.94.main]. Project: "\\mlangfs1\Public\dotnet maui\Repro Project\1185.zip"

**Comment by XamlTest** [comment]
Verified this issue with Visual Studio Enterprise 17.6.0 Preview 2.0. Repro on Windows with below Project: 
[1185.zip](https://github.com/dotnet/maui/files/11167690/1185.zip)

**Comment by kubaflo** [comment]
Thanks for the report. This thread points to upstream WinUI/VS accessibility behavior rather than an actionable MAUI backlog item, so I am closing it here. If there is a fresh MAUI-specific reproduction on current tooling, please open a new issue and link back.

**Comment by kubaflo** [comment]
Reopening — this issue should not have been closed.

It carries `s/verified` (a maintainer or partner reproduced it), `s/triaged` (it already went through triage), `partner/winui` (a partner team is tracking it). Those labels mean the report was already confirmed as a real product defect, so it is not eligible for backlog cleanup closure. Closure here requires specific fix evidence (a merged PR that closes it, or a `fixed-in-*` label), and there was none.

Sorry for the noise, and thanks to everyone who flagged it.

> Disclosure: this comment was generated by AI.
